app.js

```diff
@@ -43,6 +47,39 @@ MongoDB.once('open', function() {
     console.log('mongodb connection opened');
 });
 
+// initialize middleware for user auth
+
+app.use(express.static('public'));
+app.use(cookieParser);
+app.use(bodyParser);
+app.use(session({ secret: 'lounge-secret' }));
```

Comment:
There are some other settings we want to use here. Check out that screenshot I sent you again, @EvanDWalsh . We need to create a new Redis session object.